### ISSUE TITLE
make docker file

### ISSUE THREAD

**Issue by didillysquat** [didillysquat/SymPortal_framework#54]
I think @HughWarrington may have already made a Dockerfile for this. If he still has it I can have a look at it and adapt to the latest version of the SP code.

**Comment by HughWarrington** [comment]
Yeah I wrote one, it's on my other machine though so I won't be able to get it until I'm back next week!

**Comment by didillysquat** [comment]
@HughWarrington , I'm going to take this on myself (as discussed). Best I learn how :).

**Comment by HughWarrington** [comment]
Good idea, I think the Dockerfile I made is way outdated now, and possibly
lost...!
